pkg/lang/python/aws_runtime/aws.go

```diff
@@ -100,6 +100,19 @@ func (r *AwsRuntime) AddExecRuntimeFiles(unit *core.ExecutionUnit, result *core.
 		dockerFile = dockerfileLambda
 		dispatcher = dispatcherLambda
 		requirements = execRequirementsLambda
+
+		python.AddRequirements(unit, fsRequirements)
```

Comment:
If only JS could do this 😢

Reply:
yeah definitely a place of improvement for us when we get the time to imrpove node